var/spack/repos/builtin/packages/clamr/package.py

```diff
@@ -0,0 +1,89 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Clamr(CMakePackage):
+    """The CLAMR code is a cell-based adaptive mesh refinement (AMR)
+       mini-app developed as a testbed for hybrid algorithm development
+       using MPI and OpenCL GPU code.
+    """
+
+    homepage = "https://github.com/lanl/CLAMR"
+    url      = ""
+    tags     = ['proxy-app']
+
+    version('master', git='https://github.com/lanl/CLAMR.git')
+
+    variant(
+        'nographics', default=False,
+        description='Build without graphics')
+    variant(
+        'opengl', default=True,
+        description='Build with OpenGL or MPE')
+    variant('debug', default=False, description='Debugging enabled')
+    variant('release', default=False, description='Release version')
+    variant('full', default=False, description='full double precision')
+    variant(
+        'mixed', default=False,
+        description='intermediates double, arrays single precision')
+    variant('single', default=False, description='single precision')
+
+    depends_on('mpi')
+    depends_on('mpe', when='-opengl')
```

Comment:
What @alalazo means is that most of the time, we use `~` in packages to denote "without" instead of `-`. I think both should work though...